<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cronoacqDlg</class>
 <widget class="QMainWindow" name="cronoacqDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>cronoacq</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="nofclients">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>no of clients: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="propertydockWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>DAQ settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QtTreePropertyBrowser" name="variantEditor" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="LogdockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>274</width>
     <height>139</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextBrowser" name="logBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstart_acquistion"/>
   <addaction name="actionrestart_acquisition"/>
   <addaction name="actionstop_acquisition"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstart_sampling"/>
   <addaction name="actionsingle_sampling"/>
   <addaction name="actionpause_sampling"/>
  </widget>
  <widget class="QDockWidget" name="TracedockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Trace widget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="resetgraphButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-undo">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionstart_acquistion">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/startacq.svg</normaloff>:/icons/startacq.svg</iconset>
   </property>
   <property name="text">
    <string>start acquistion</string>
   </property>
  </action>
  <action name="actionrestart_acquisition">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/restartacq.svg</normaloff>:/icons/restartacq.svg</iconset>
   </property>
   <property name="text">
    <string>restart acquisition</string>
   </property>
  </action>
  <action name="actionstop_acquisition">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/stopacq.svg</normaloff>:/icons/stopacq.svg</iconset>
   </property>
   <property name="text">
    <string>stop acquisition</string>
   </property>
  </action>
  <action name="actionstart_sampling">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/play.svg</normaloff>:/icons/play.svg</iconset>
   </property>
   <property name="text">
    <string>start sampling</string>
   </property>
  </action>
  <action name="actionsingle_sampling">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/playsingle.svg</normaloff>:/icons/playsingle.svg</iconset>
   </property>
   <property name="text">
    <string>single_sampling</string>
   </property>
  </action>
  <action name="actionpause_sampling">
   <property name="icon">
    <iconset resource="cronoacq_ressources.qrc">
     <normaloff>:/icons/pause.svg</normaloff>:/icons/pause.svg</iconset>
   </property>
   <property name="text">
    <string>pause_sampling</string>
   </property>
  </action>
  <zorder>propertydockWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>resetgraphButton</tabstop>
  <tabstop>logBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="cronoacq_ressources.qrc"/>
 </resources>
 <connections/>
</ui>
